core/src/listener/requestResponnse-L2-request.ts

```diff
@@ -0,0 +1,80 @@
+import { L2Endpoint__factory } from '@bisonai/orakl-contracts'
```

Comment:
Typo in the name of the file. Additionally, it has different format than data-feed-L2.ts. How about making it request-response-L2-request.ts?

Reply:
Reporter and Worker follow this style as well.